src/effects/Echo.cpp

```diff
@@ -163,3 +188,29 @@ EffectEcho::PopulateOrExchange(ShuttleGui & S, EffectSettingsAccess& access)
    S.EndMultiColumn();
    return nullptr;
 }
+
+
+bool EffectEcho::Validator::ValidateUI()
+{
+   mAccess.ModifySettings
+   (
+      [this](EffectSettings& settings)
+   {
+      // pass back the modified settings to the MessageBuffer
+      EffectEcho::GetSettings(settings) = mSettings;
```

Comment:
Do NOT add these lines in the second commit.  Only at the last commit.

Reply:
I commented it out + put a TODO comment "uncomment at final step"